packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -516,4 +522,29 @@ router.get(
   })
 );
 
+router.post(
+  "/trigger-update",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const patients = await getPatients({ cxId });
+    const chunks = chunk(patients, patientChunkSize);
+
+    for (const chunk of chunks) {
```

Comment:
It would also be good to log the amount of patients, the amount of chunks and, at the end, a confirmation of how many were been updated.